.rubocop.yml

```diff
@@ -1,8 +1,11 @@
 inherit_gem:
   ezcater_rubocop: conf/rubocop_gem.yml
 
+plugins:
+  - rubocop-capybara
```

Comment:
Just curious, why are we adding `rubocop-capybara`? :thinking:

Reply:
@thiagopecanha I was hitting the error described in [this slack thread](https://ezcater.slack.com/archives/C94MF14JF/p1745523988511109).